solr/core/src/java/org/apache/solr/handler/admin/ZookeeperInfoHandler.java

```diff
@@ -529,18 +529,13 @@ boolean printTree(JSONWriter json, String path) throws IOException {
         label = idx > 0 ? path.substring(idx + 1) : path;
       }
       json.startObject();
-      //writeKeyValue(json, "data", label, true );
-      json.writeString("data");
-      json.writeNameSeparator();
-
-      json.startObject();
-      writeKeyValue(json, "title", label, true);
+      writeKeyValue(json, "text", label, true);
       json.writeValueSeparator();
-      json.writeString("attr");
+      json.writeString("a_attr");
       json.writeNameSeparator();
       json.startObject();
-      writeKeyValue(json, "href", "admin/zookeeper?detail=true&path=" + URLEncoder.encode(path, "UTF-8"), true);
-      json.endObject();
+      String href = "admin/zookeeper?detail=true&path=" + URLEncoder.encode(path, StandardCharsets.UTF_8);
+      writeKeyValue(json, "href", href, true);
```

Comment:
The `jstree` JSON data format changed so it requires a different structure. It wasn't easy to modify this structure for ZK since it came directly from the `ZookeeperInfoHandler`. I looked and didn't see other uses of this, so it looked safe to change here.

@ErickErickson any concerns?